src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -0,0 +1,1046 @@
+        if (a == null && b == null) {
+            return null;
+        } else if (a == null) {
+            return b;
+        } else if (b == null) {
+            return a;
+        }
+        return Float.sum(a, b);
+    }
+
+    /**
+     * Retrieve basic system stats - auto answering feature, cameras/mics basic info etc.
+     *
+     * @param statistics map to set data to
+     * @throws Exception during http communication, except 403
+     */
+    private void retrieveSystemStatus(Map<String, String> statistics) throws Exception {
+        ArrayNode response = doGet(STATUS, ArrayNode.class);
+        response.iterator().forEachRemaining(jsonNode -> {
+            String langtag = getJsonProperty(jsonNode, "langtag", String.class);
+            ArrayNode stateList = getJsonProperty(jsonNode, "stateList", ArrayNode.class);
+
+            if (langtag != null && stateList != null) {
+                statistics.put("System Status#" + normalizeStringPropertyName(
+                        langtag.replaceAll("_", " ")),
+                        stateList.get(0).asText().replaceAll("_", " ").toUpperCase());
+            }
+        });
+    }
+
+    /**
+     * Change property name to start with a capital letter
+     *
+     * @param string parameter name
+     * @return {@link String} property name with first capital letter
+     */
+    private String normalizeStringPropertyName(String string) {
+        return string.substring(0, 1).toUpperCase() + string.substring(1).toLowerCase();
+    }
+
+    /**
+     * Retrieve the current state of the collaboration session
+     *
+     * @return Integer value of the current device volume level
+     * @throws Exception during http communication
+     */
+    private void retrieveCollaborationStatus(Map<String, String> statistics) throws Exception {
+        JsonNode response = doGet(COLLABORATION, JsonNode.class);
+        if (!response.isNull()) {
+            String sessionState = getJsonProperty(response, "state", String.class);
+            statistics.put("Collaboration#Session State", sessionState);
+            if ("ACTIVE".equals(sessionState)) {
+                statistics.put("Collaboration#Session ID", getJsonProperty(response, "id", String.class));
+            }
+        }
+    }
+
+    /**
+     * Retrieve current volume level of the device
+     *
+     * @return Integer value of the current device volume level
+     * @throws Exception during http communication
+     */
+    private Integer retrieveVolumeLevel() throws Exception {
+        return doGet(VOLUME, Integer.class);
+    }
+
+    /**
+     * Update device's volume with a given value
+     *
+     * @param value target value for the device volume
+     * @return boolean value indicating the success of the operation
+     * @throws Exception during http communication
+     */
+    private boolean updateVolumeLevel(int value) throws Exception {
+        return doPost(VOLUME, value).getStatusCode().is2xxSuccessful();

```

Comment:
Interesting that this is the only one call checking response status code. If response code is not verified by the base class(es), they we have potential issue with all other calls. If it is verified by based class(es), why this specific call needs to verify it again?

Reply:
This specific http endpoint does not return any value, so the only way to see if it has succeeded is to check it's status code. Adding this to javadoc